electron/main.ts

```diff
@@ -461,9 +461,20 @@ ipcMain.handle('kill', async (event, appName, runner) => {
   return Game.get(appName, runner).stop()
 })
 
-ipcMain.handle('checkDiskSpace', async (event, folder) => {
-  const { free, size: diskSize } = await checkDiskSpace(folder)
-  return `${getFileSize(free)}/${getFileSize(diskSize)}`
+ipcMain.handle('checkDiskSpace', async (event, folder: string) => {
+  if (folder.startsWith('/')) {
```

Comment:
just wondering, does this work on windows? looks like unix specific

Reply:
yes, won't work. I will figure out another way of checking that.

Reply:
So actualy the function `checkDiskSpace` checks if the path is valid based on the platform so we dont need to do it ourselves.
So I just changed it to a try catch now.